modules/renter/hostdb/update.go

```diff
@@ -29,6 +29,23 @@ func findHostAnnouncements(b types.Block) (announcements []modules.HostDBEntry)
 	return
 }
 
+// updateHistoricInteractions adds the recent interactions to the historic interactions
+// and progresses the decay
+func (hdb *HostDB) updateHistoricInteractions() {
+	hosts := hdb.hostTree.All()
+	for _, host := range hosts {
```

Comment:
I'm wondering if this loop is going to be too slow when we have more hosts. Iterating through 1000 objects once per block is not so bad, but perhaps it'd be better to have a global decay value or something.

I ~think we are safe to leave it this way for now, but we might want to tag a TODO indicating performance concerns.